bot/bot.py

```diff
@@ -0,0 +1,50 @@
+import logging
+import os
+
+import disnake
+from disnake.ext import commands
+
+logger = logging.getLogger(__name__)
+
+try:
+    import dotenv
+    dotenv.load_dotenv()
+except ModuleNotFoundError:
+    pass
+
+DEV_LOG_CHANNEL = os.environ.get("DEV_LOG_CHANNEL")
+TOKEN = os.environ.get("TOKEN")
+
+
+class SirRobin(commands.Bot):
+    """Sir-Robin core."""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.name = "Sir-Robin"
+        self.dev_log = int(DEV_LOG_CHANNEL)
```

Comment:
This `__init__` looks unnecessary; the name isn't particularly useful, and `DEV_LOG_CHANNEL` should be cast to `int` in the first place.

Reply:
This was removed in [5395e44](https://github.com/python-discord/sir-robin/pull/10/commits/5395e4430800b14264f688b2bc09ebc375ee436c)